### DISCUSSION TITLE
반복요청시 -303 에러

### DISCUSSION THREAD

**Discussion by disabled-variable** [storycraft/node-kakao#728] [Q&A]
특정요청을 딜레이없이 반복해서 요청시 -303 에러가 발생합니다.

카카오톡 Rate_Limit에 관련되어있다고 찾아봤는데,

해결방안이 의도적으로 딜레이를 주는것 말고는 없을까요?

**Comment by benesiaxd** [comment]
네

**Reply by disabled-variable** [reply]
도배업자들은 몇백개의 폭탄패킷을 어떻게 보내는거죠?
그사람들도 303에러는 해당되지않을까요?

**Reply by benesiaxd** [reply]
불가능합니다.
